docs/advanced-features/custom-error-page.md

```diff
@@ -97,4 +97,4 @@ If you have a custom `Error` component be sure to import that one instead. `next
 
 ### Caveats
 
-- `Error` currently does not support Next.js [Data Fetching methods](/docs/basic-features/data-fetching.md) like [`getStaticProps`](/docs/basic-features/data-fetching.md#getstaticprops-static-generation) or [`getServerSideProps`](/docs/basic-features/data-fetching.md#getserversideprops-server-side-rendering).
+- `Error` currently does not support Next.js [Data Fetching methods](https://nextjs.org/docs/basic-features/data-fetching/index) like [`getStaticProps`](/docs/basic-features/data-fetching/get-static-props) or [`getServerSideProps`](/docs/basic-features/data-fetching/get-server-side-props).
```

Comment:
You dont need to add the full url 😊 and the file is `overview` => `/docs/basic-features/data-fetching/overview.md`

Reply:
Im pretty sure we add the file extension to the end of the path too

Reply:
Yeah, I might have missed it then. 🙈